app/serializers/service_provider_serializer.rb

```diff
@@ -31,10 +32,18 @@ def agency_id
     object&.agency&.id
   end
 
-  def certs
-    object.certificates.map(&:to_pem)
+  # The one expiring latest - no way to know which was added last. Since this is
+  # temporary I think it's ok.
+  def cert
+    object.certificates.sort do |cert_a, cert_b|
+      cert_b.not_after - cert_a.not_after
+    end.first.to_pem
```

Comment:
```suggestion
    object.certificates.sort_by(&:not_after).last
```

Reply:
🤦

Reply:
Oh I know what happened - I thought that I needed to call `to_pem` before I could call `not_after` so that's why I needed the block, but then when I figured that out I forgot to switch it. Thanks!